src/server/src/Server.cpp

```diff
@@ -210,29 +253,25 @@ std::string Server::stop()
     return "OK";
 }
 
-
 std::string Server::quit()
 {
     set_state(State::Close);
 
     return "OK";
 }
 
-
 void Server::enable_position_control()
 {
     drdRegulatePos(true);
     drdEnableFilter(false);
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
done